gfx/RetroDaysTileset/tile_config.json

```
@@ -7808,20 +7809,20 @@
       "sprite_offset_x": -5,
       "sprite_offset_y": -10,
       "tiles": [
-        { "id": "mon_dragon_dummy", "fg": 3040, "rotates": false },
-        { "id": "mon_tyrannosaurus", "fg": 3041, "rotates": false },
-        { "id": "mon_triceratops", "fg": 3042, "rotates": false },
-        { "id": "mon_fungaloid_queen", "fg": 3043 },
-        { "id": "mon_fungaloid_seeder", "fg": 3044 },
-        { "id": "mon_fungaloid_tower", "fg": 3045 },
-        { "id": "mon_skeleton_hulk", "fg": 3046, "rotates": false },
-        { "id": "mon_zombie_kevlar_2", "fg": 3047, "rotates": false },
-        { "id": "mon_zombie_hulk_pk", "fg": 3048, "rotates": false },
-        { "id": "mon_zombie_hulk", "fg": 3048, "rotates": false },
-        { "id": "overlay_mutation_THRESH_URSINE", "fg": 3049 },
-        { "id": "overlay_mutation_THRESH_CATTLE", "fg": 3050 },
-        { "id": "overlay_male_mutation_THRESH_CATTLE", "fg": 3051 },
-        { "id": "overlay_mutation_THRESH_LIZARD", "fg": 3052 }
+        { "id": "mon_dragon_dummy", "fg": 3041, "rotates": false },
```

You actually need to change these by 16, not just one.
```suggestion
        { "id": "mon_dragon_dummy", "fg": 3057, "rotates": false },
```

---

Ohh, that's how it works. Got it, thanks. 5 min.

---

Wait, are you sure it's `3057` and not `3056`?

---

Assuming you meant `3056`. Done.